wallet/caip2.md

```diff
@@ -0,0 +1,54 @@
+---
+namespace-identifier: wallet-caip2
+title: Wallet Namespace - RPC namespaces
+author: ["Vandan Parikh (@vandan)", "Bumblefudge (@bumblefudge)"]
+discussions-to: <URL of PR, mailing list, etc>
+status: Draft
+type: <Standard | Meta | Informational>
+created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+requires (*optional): <["CAIP-X", "CAIP-Y"]>
+replaces (*optional): <CAIP-Z>
+---
+
+*For context, see the [CAIP-2][] specification.*
+
+Each `chainId` reference in the `wallet:` namespace is defined as the namespace identifier for another namespace.
+
+This combination (i.e. `wallet:eip155`) refers not to an `eip155` network but specifically to an application<>wallet connection defined according to the wallet and RPC conventions of the `eip155` namespace.
+
+## Rationale
+
+Implementers of [CAIP-25][] and other multichain application<>wallet connections have long struggled with how to map offchain or direct-to-wallet methods, notifications, and metadata to one another across namespaces, and how to safely and ergonomically segregate them from node-dependent/network-dependent functions.
+For this reason, an abstraction (in the form of a "meta-namespace" routing to the other namespaces) is here proposed.
+
+## Semantics
+
+`wallet:{namespace}` is functionally an alias for `{namespace}:wallet`, where `:wallet` refers to a special-case of a wallet connection, not a proper network (as required by the CAIP-2 profile for `eip155`).
```

Comment:
sorry for late review. My understanding of introducing this special `wallet` meta namespace was so that we could avoid the usage of `:wallet` entirely. This sentence seems to imply that `{namespace}:wallet` is also valid and interchangeable with `wallet:{namespace}`. Perhaps I am reading this wrong?

Reply:
Yeah I agree this is a bit confusingly phrased and sortof implies (at least to my ear) that `{namespace}:wallet` is an established pattern that we're providing an alternative syntax for...